src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITEngine.cpp

```diff
@@ -152,7 +62,7 @@ std::unique_ptr<LLVMJITEngine> LLVMJITEngineBuilder::build() {
       .setErrorStr(&error);
 
   auto engine = std::make_unique<LLVMJITEngine>();
-  engine->engine = eb.create(buildTargetMachine());
+  engine->engine = eb.create(tm_.release());
```

Comment:
why tm_ as unique ptr?

Reply:
Just to make ownership clear.

Reply:
who has the ownership after release?

Reply:
![image](https://user-images.githubusercontent.com/47744603/209598984-99b0bcba-cabd-4ef3-b320-f888dedc28ca.png)
```llvm::EngineBuilder``` will take it.